### DISCUSSION TITLE
Run ComfyUI without having to run Krita

### DISCUSSION THREAD

**Discussion by SyamsQ** [Acly/krita-ai-diffusion#777] [Q&A] [answered]
I installed ComfyUI via Krita. Is there a way to run ComfyUI without having to run Krita?

[a comment by an automated account is not shown]

**Reply by SyamsQ** [reply]
Guide me, it seems doesn't run ![image](https://github.com/Acly/krita-ai-diffusion/assets/17028491/f6cb9e86-d71a-4963-be3a-43718b0e5e78)

**Reply by SyamsQ** [reply]
Great! It's work. But, why we can't update it?
![image](https://github.com/Acly/krita-ai-diffusion/assets/17028491/ae1cfd55-7d68-466a-89f5-34b130c40b98)

**Reply by Hypersonic-Popcorn** [reply]
If you want an independent ComfyUI, it's probably best to install it from the instructions on https://github.com/comfyanonymous/ComfyUI and then follow the instructions here https://github.com/Acly/krita-ai-diffusion/wiki/ComfyUI-Setup. The good news is that you can just move/link the models to your new install. Then you'll be able use Comfy standalone.

I've barely started with this plugin but I have a separate Comfy install and I'm able to use it as long as I'm not actively generating in Krita.

**Reply by SyamsQ** [reply]
> If you want an independent ComfyUI, it's probably best to install it from the instructions on https://github.com/comfyanonymous/ComfyUI and then follow the instructions here https://github.com/Acly/krita-ai-diffusion/wiki/ComfyUI-Setup. The good news is that you can just move/link the models to your new install. Then you'll be able use Comfy standalone.
> 
> I've barely started with this plugin but I have a separate Comfy install and I'm able to use it as long as I'm not actively generating in Krita.

Sometimes, I need to use ComfyUI features, but they don't appear in ComfyUI Krita. Therefore, I need to open ComfyUI without Krita. But, it's not uncommon for me to use both Krita and ComfyUI.

**Reply by Acly** [reply]
Starting the plugin's Comfy install directly isn't a problem. But it's good advice to use your own custom ComfyUI install if you want to modify it (update or install nodes). The one installed & managed by the plugin will be reset when there is an update.

**Reply by SyamsQ** [reply]
> Starting the plugin's Comfy install directly isn't a problem. But it's good advice to use your own custom ComfyUI install if you want to modify it (update or install nodes). The one installed & managed by the plugin will be reset when there is an update.

Can  models orcheckpoints of ComfyUI Krita and independent ComfyUI's be combined in one directory folder?

[a comment by an automated account is not shown]
